src/main.cpp

```diff
@@ -33,8 +36,38 @@ void loop()
 	for (int j = 0; j < 255; j++)
 	{
 		for (int i = 0; i < NUM_LEDS; i++)
-			leds[i] = CHSV((i+j)%254, 254, 254);
+			leds[i] = CHSV((i+j)%256, 255, 64);
 		FastLED.show();
 		delay(32);
 	}
+}
+
+void rainbowStrobe()
+{
+	int spot_brightness = 0;
+
+	for (int j = 0; j < 2048; j++)
+	{
+		for (int i = 0; i < NUM_LEDS; i++)
+		{
+			//This needs fixing up, while the colours step once per cycle,
+			//The brighness will make multiple steps.
+			//There's no reason why the brighness can't change, for example,
+			//four times faster than the colour does, but I'm lazy so here we are
+			spot_brightness = (i+(j))%256;
+			if (spot_brightness > 128)
+				spot_brightness = 128 - (spot_brightness - 128);
+			if (spot_brightness < 32)
+				spot_brightness = 32;
+			leds[i] = CHSV((i+(j/8))%256, 255, spot_brightness);			
+		}
+		FastLED.show();
+		delay(32);
+	}
+}
+
+
+void loop()
+{
+	rainbowStrobe();
```

Comment:
Maybe add some code where you can pick effects? Or just leave the other function here commented out?

Reply:
Yep, adding picker and comments